docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -0,0 +1,238 @@
+FreeIPA
```

Comment:
Please add `:orphan:` at the top so that people can choose not to pull it in.

Reply:
Done. I don't think we need to add this article to any index in the blueprint, right?

Reply:
I do think you should add it to the index in the blueprint so when I build the blueprint I can browse to it and look at it.